common/src/main/java/net/adoptopenjdk/icedteaweb/OutputUtils.java

```diff
@@ -0,0 +1,25 @@
+package net.adoptopenjdk.icedteaweb;
+
+import java.io.PrintWriter;
+import java.io.StringWriter;
+
+public class OutputUtils {
+
+    public static String exceptionToString(Throwable t) {
+        if (t == null) {
+            return null;
+        }
+        String s = "Error during processing of exception";
```

Comment:
Assignment is redundant
if assignment is removed the variable can be made final

Reply:
refactored similar as apache.commons.lang.exception ExceptionUtils does it